policy/modules/services/console_setup.te

```diff
@@ -0,0 +1,42 @@
+policy_module(console_setup, 1.0)
+
+########################################
+#
+# Declarations
+#
+
+type console_setup_t;
+type console_setup_exec_t;
+init_daemon_domain(console_setup_t, console_setup_exec_t)
+
+type console_setup_conf_t;
+files_config_file(console_setup_conf_t)
+
+type console_setup_runtime_t;
+files_pid_file(console_setup_runtime_t)
+
+########################################
+#
+# Local policy
+#
+
+allow console_setup_t self:capability sys_tty_config;
+allow console_setup_t self:fifo_file rw_inherited_fifo_file_perms;
+
+corecmd_exec_bin(console_setup_t)
+corecmd_exec_shell(console_setup_t)
```

Comment:
These two corecmd entries need to be moved below `manage_files_pattern(console_setup_t, console_setup_conf_t, console_setup_conf_t)` as per the style rules.